tests/certification/secretstores/local/file/components/defaultnestedseparator/localsecrets.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: file-secret-store
+spec:
+  type: secretstores.local.file
+  version: v1
+  metadata:
+  - name: secretsFile
+    value: ./components/secrets.json
+  - name: nestedSeparator
```

Comment:
Should we remove setting `nestedSeparator` from here, so that we are able to verify the default behavior, without explicitly setting it.

Reply:
makes sense!